lib/documents/schemas/drug_safety_updates.json

```diff
@@ -29,6 +29,7 @@
       "filterable": true,
       "allowed_values": [
         {"label": "Anaesthesia and intensive care", "value": "anaesthesia-intensive-care"},
+        {"label": "Breasfeeding", "value": "breasfeeding"},
```

Comment:
I'm assuming this should be "Breastfeeding" and "breastfeeding" respectively (missing t)

Reply:
Ah, I copied directly from the Zendesk ticket and hadn't noticed the typo, thank you! 🦅👁